docs/Deep-Dives/metaapi.md

```diff
@@ -0,0 +1,200 @@
+
+A "Google Spreadsheet Append" connector is responsible for appending the values to a selected spreadsheet.
+
+![Add Google Spreadsheet API to Spell](images/metaapi/meta-6.png){: .border}
+
+![Choose the "append" endpoint (Connector)](images/metaapi/meta-7.png){: .border}
+
+Next, click "Continue with 2 connectors" and go through the following steps with values to create your Spell.
+
+![Submit your spell](images/metaapi/meta-8.png){: .border}
+
+Now you should see the edit Spell view.
+
+![Edit spell view](images/metaapi/meta-9.png){: .border}
+
+
+## 6. Authenticate your Connectors
+
+As you can see in the right sidebar, there are a few alerts. 
+To update our configuration, click the "Fix" button next to Flotiq User API.
+
+![Fix authentication problem](images/metaapi/meta-10.png){: .border}
+
+In the settings panel for Flotiq User API, click "Configure Authentication",
+and paste here your API Key from Flotiq
+
+!!! Note
+    If you are new to Flotiq, check how to obtain your [Flotiq API Key](https://flotiq.com/docs/API/)
+
+![Add your Flotiq API Key to the Connector configuration](images/metaapi/meta-11.png){: .border}
+
+Approve your settings and go back to the errors list clicking "x" in the sidebar.
+
+![Close Flotiq Connector settings](images/metaapi/meta-13.png){: .border}
+
+The next step is to authenticate the Google Spreadsheet connector.
+Click "Fix" next to "Google Spreadsheet".
+
+![Fix Google Spreadsheet issues](images/metaapi/meta-14.png){: .border}
+
+This configuration is slightly different from Flotiq because it uses oAuth2.0. 
+Use the "One-click Auth with Meta Api" wizard to authenticate.
+
+![Run step-by-step Authentication wizard](images/metaapi/meta-15.png){: .border}
+
+There are also a few properties that should be updated in Google Spreadsheet Connector settings. 
+
+Please set:
+- `spreadsheetId` value to spreadsheet id obtained in the first step,
+- `valueInputOption` to `USER_ENTERED`,
+- `range` to `A2`.
+
+![Update Google Spreadsheet connector settings](images/metaapi/meta-16.png){: .border}
+
+We are almost ready!
+
+## 7. Add code to process data
+
+In the previous steps, we were focused on the sidebar in the spell configuration view.
+Now let's move to the section with code.
+
+It is up to you what transformations you will create here. 
+In our case, we will sum up all product storage quantities and place them next to the current date.
+
+So, update the code - add this snippet below `//#endregion End of Flotiq User API connector`:
+
+```javascript
+const dataRow = [
+    (new Date()).toLocaleString(), // date
+    flotiqUserApiResult.data.reduce((sum, prod) => (sum + prod.stock_quantity), 0) // quantity
+];
+```
+
+And in the Google Spreadsheet connector code replace null with our dataRow:
+```javascript
+	values: [
+		dataRow
+	]
+```
+
+![Modify connector code](images/metaapi/meta-17a.png){: .border}
+
+That's it! It's so simple to code your custom transformations. Ready to test?
```

Comment:
I like the connection with the reader :)

Reply:
:)